### PR TITLE
PSVAMB-7641 Entry Vendor Task Finished HTTP notification

### PR DESCRIPTION
**Pull request by yusitnikov**
- add "Entry Vendor Task Finished" HTTP notification (similar to the existing email notification: https://github.com/kaltura/server/blob/Orion-15.1.0/deployment/updates/scripts/xml/notifications/2018_02_22_entry_vendor_task_done.template.xml)
- add an option to choose entry vendor task as an option for HTTP notification

### CODE REVIEW

**Comment by yusitnikov** [comment]
The related server-saas-config PR: https://github.com/kaltura/server-saas-config/pull/1710

**Comment by MosheMaorKaltura** [comment]
reviewed

File: deployment/updates/scripts/xml/notifications/2019_05_26_httpEntryVendorTaskDone.template.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                <eventObjectType>42</eventObjectType> <!-- EventNotificationEventObjectType::ENTRY_VENDOR_TASK -->
+                <eventConditions objectType="array">
+                    <item objectType="KalturaEventFieldCondition">
+                        <field objectType="KalturaEvalBooleanField">
+                            <code>$scope->getObject() instanceof EntryVendorTask &amp;&amp; in_array(EntryVendorTaskPeer::STATUS, $scope->getEvent()->getModifiedColumns())</code>
+                        </field>
+                    </item>
+                    <item objectType="KalturaEventFieldCondition">
+                        <field objectType="KalturaEvalBooleanField">
+                            <code>$scope->getObject()->getStatus() == EntryVendorTaskStatus::READY</code>
```
Comment (yossipapi) [inline deployment/updates/scripts/xml/notifications/2019_05_26_httpEntryVendorTaskDone.template.xml]: In the current email we do not send notifications for machine task, this is based on product request.

Either add the missing condition or make it configurable when deploying the notification.

Comment (yusitnikov) [reply deployment/updates/scripts/xml/notifications/2019_05_26_httpEntryVendorTaskDone.template.xml]: I removed the condition for creation mode on purpose - the HTTP notification should be triggered in all cases

**Pull request merged**